plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/jdt/ui/wizards/NewInterfaceCreationWizardPage.java

```diff
@@ -0,0 +1,196 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat, Inc and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat, Inc - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.reddeer.eclipse.jdt.ui.wizards;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.reddeer.common.exception.RedDeerException;
+import org.eclipse.reddeer.core.reference.ReferencedComposite;
+import org.eclipse.reddeer.jface.wizard.WizardPage;
+import org.eclipse.reddeer.swt.api.TableItem;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.swt.impl.table.DefaultTable;
+import org.eclipse.reddeer.swt.impl.text.DefaultText;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+
+/**
+ * Wizard page for creating a java interface.
+ * 
+ * @author zcervink@redhat.com
+ * 
+ */
+public class NewInterfaceCreationWizardPage extends WizardPage {
```

Comment:
I am missing method for setting modifiers (as @odockal pointed out in one of his previous comments).

Reply:
Working on that, just a minute.

Reply:
Added and fixed.